woocommerce/woocommerce-ios
# Login: Fix layout issue on account mismatch error screen

Closes: #8061 


### Description

The contents of the account mismatch screen is currently truncated on small devices. This PR fixes that by adding a few tweaks:
- Wrap all contents inside a scrollview
- Shorten the CTA on the primary button to keep it concise.

### Testing instructions

- Build and run the app on a small screened simulator.
- Skip login onboarding and select Enter site address.
- Enter your test site address and log in with a WP.com account that is not associated with that site.
- Notice that the account mismatch screen is then displayed, and the view should be scrollable. The title of the primary button should not be truncated.

Please feel free to test on a larger device to ensure that everything still looks great.

### Screenshots

Before on iPhone SE simulator
<img src="https://user-images.githubusercontent.com/5533851/200999268-afa5195b-1877-444b-a27b-c8897952014f.png" width=320 />
After:
https://user-images.githubusercontent.com/5533851/200999337-8152bb9a-265a-427d-9aa8-dfbdb67be6d3.mp4

On iPhone 14 Pro Max simulator:

<img src="https://user-images.githubusercontent.com/5533851/200999400-9e4b264f-011d-45fb-b12d-87a4b20bb941.png" width=320 />




---
